workers/unity/Packages/com.improbable.gdk.core/Worker/WorkerConnector.cs

```diff
@@ -10,17 +10,33 @@
 
 namespace Improbable.Gdk.Core
 {
+    /// <summary>
```

Comment:
All the stuff in this that is virtual should tell a user what is expected of them if they override it.

Reply:
Is that not satisfied by the summary + return description?